README.md

```diff
@@ -5,12 +5,34 @@ and provides request forwarding as a routing convenience. It can be used to
 shard your application in a way that's scalable and fault tolerant.
 
 # Installation
-To install Ringpop:
+To install Ringpop for usage as a library:
 
 ```
 npm install ringpop
 ```
 
+Prepare the current directory for development:
```

Comment:
I think this is a bit confusing. We should make clear distinction between Installation as a User and Setup for a Developer. The way the `npm install ringpop` and `npm install` instructions flow makes it seem like both are required.

I typically see these broken into 2 sections:

```
# Installation
```

and 

```
# Developing
```